benchmarks/hash.js

```diff
@@ -22,19 +22,22 @@ const algs = [
   'keccak-384',
   'keccak-512',
   'murmur3-32',
-  'murmur3-128'
+  'murmur3-128',
+  'dbl-sha2-256',
+  'blake2b-256',
+  'blake2b-512',
+  'blake2s-256'
 ]
 
 algs.forEach((alg) => {
   suite.add(alg, function (d) {
     const buf = Buffer.alloc(10 * 1024)
     buf.fill(Math.ceil(Math.random() * 100))
-
-    multihashing(buf, alg, (err, res) => {
-      if (err) throw err
-      list.push(res)
-      d.resolve()
-    })
+    multihashing(buf, alg)
+      .then(res => {
+        list.push(res)
+        d.resolve()
+      })
```

Comment:
This is going to cause an unhandledRejection in case of errors.

Reply:
fixed